apps/modernization-ui/src/apps/search/patient/add/layout/AddPatientLayout.spec.tsx

```diff
@@ -0,0 +1,113 @@
+import { render } from '@testing-library/react';
+import { AddPatientLayout } from './AddPatientLayout';
+import { NavSection } from 'design-system/inPageNavigation/InPageNavigation';
+
+class MockIntersectionObserver {
+    observe = jest.fn();
+    unobserve = jest.fn();
+    disconnect = jest.fn();
+    constructor(callback: (entries: IntersectionObserverEntry[]) => void) {
+        callback([{ isIntersecting: true, target: { id: 'section1' } } as IntersectionObserverEntry]);
+    }
+}
+
+Object.defineProperty(window, 'IntersectionObserver', {
+    writable: true,
+    configurable: true,
+    value: MockIntersectionObserver
+});
```

Comment:
suggestion:  if you're not testing the behavior of the Intersection Observer you don't have to mock it, you can mock `useInPageNavigation`